package.json

```diff
@@ -107,7 +107,7 @@
     "babel-loader": "^8.2.2",
     "copy-webpack-plugin": "^9.0.1",
     "css-loader": "5.2.6",
-    "electron": "^14.0.0",
+    "electron": "^13.1.7",
```

Comment:
Why not 13.3.0 ? (is the issue present on that version?)

Reply:
This change is made by reverting [the diff for the line of electron version update](https://github.com/FreeTubeApp/FreeTube/commit/0db6dd1f970747e34336e964245350cbfdcf7f21#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L110)

I can make it `^13.3.0` but it's almost the same as `^13.1.7`
- `^13.3.0`: `>=13.3.0 <14.0.0`
- `^13.1.7`: `>=13.1.7 <14.0.0`

If you look at https://github.com/FreeTubeApp/FreeTube/pull/1658/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519R6721
`electron`'s version is updated to `13.3.0` anyway (or the latest 13.x if there were newer versions)

Reply:
Let's go ahead and update it to `13.3.0` assuming it doesn't cause the same issues with `14.0.0`. Outside of that, I have no issues with moving forward with this.